src/assemble_workflow/dist.py

```diff
@@ -13,18 +13,20 @@
 import zipfile
 from abc import ABC, abstractmethod
 
-from assemble_workflow.bundle_rpm import BundleRpm
+from assemble_workflow.bundle_linux_distro import BundleLinuxDistro
 from manifests.build_manifest import BuildManifest
 from system.zip_file import ZipFile
 
 
 class Dist(ABC):
+
     def __init__(self, name: str, path: str, min_path: str, build_cls: BuildManifest.Build) -> None:
         self.build_cls = build_cls
         self.name = name
         self.filename = name.lower()
         self.path = path
         self.min_path = min_path
+        self.bundle_linux_distro = BundleLinuxDistro(self.filename, self.path, self.min_path)
```

Comment:
This shouldn't be in the base class, a Windows distribution wouldn't know anything about linux.